cypress/support/step-definitions/textbox-steps.js

```diff
@@ -117,21 +117,21 @@ Then('Multiple label is set to {word}', (text) => {
 });
 
 Then('Textbox component is labelInline', () => {
-  label().should('have.css', TEXT_ALIGN, 'left');
+  label().parent().should('have.css', TEXT_ALIGN, 'flex-start');
```

Comment:
The same is here: `flex-start` make it as a const? And use in all test

Reply:
fixed